sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/firestore/FirestoreOptions.java

```diff
@@ -66,13 +66,13 @@ public interface FirestoreOptions extends PipelineOptions {
    */
   @Description("Firestore endpoint (host and port)")
   @Default.String("batch-firestore.googleapis.com:443")
-  String getHost();
```

Comment:
Is this ok to do these breaking changes? Do we need to mention this in CHANGES.md?

Reply:
yeah let me add another item to CHANGES.md